<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="confirmer">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>630</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>CryptoPlus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 28pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 28pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Méthode</string>
   </property>
  </widget>
  <widget class="QLabel" name="deca">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 28pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Décalage</string>
   </property>
  </widget>
  <widget class="QLabel" name="cle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 28pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Clé de Décryptage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delet">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>630</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Effacer</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-140</x>
     <y>370</y>
     <width>671</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="carre">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>-10</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Carré de Polybe</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="vigenere">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Code de Vigenère</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cesar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Code de César</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ROT">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ROT13</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="test">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>260</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="dec">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Décryptage</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cry">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Cryptage</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="test1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="message">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>511</width>
     <height>151</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Entrez le texte à crypter/décrypter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultat">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Résultat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cdeca">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="verification">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Calibri&quot;;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cle_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cle_decryptage">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>600</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cle_decryptage_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
